enterprise-suite/templates/es-console.yaml

```diff
@@ -14,6 +14,25 @@ spec:
       securityContext:
         runAsUser: {{ .Values.podUID }}
       {{ end }}
+
+      initContainers:
+      - name: nginx-conf-envsubst
+        image: alpine
+        resources:
+          requests:
+            cpu: {{ default .Values.defaultCPURequest }}
+            memory: {{ default .Values.defaultMemoryRequest }}
+        command:
+        - /bin/sh
+        - -c
+        args:
+        - apk add gettext libintl && cat /etc/es-console/default.conf.tmpl | envsubst > /etc/nginx/conf.d/default.conf
```

Comment:
Just curious.  How is the environment initially setup so that `envsubst` has something to work with?